.github/workflows/main.yml

```diff
@@ -71,7 +71,7 @@ jobs:
 
       # Install / Prepare
       - name: "Add EE Composer repository"
-        run: "composer -n config repositories.ee vcs ssh://git@distribution.akeneo.com:443/pim-enterprise-dev-flagbit-gmbh-co-kg.git"
+        run: "composer -n config repositories.ee vcs ${{ secrets.SSH_AKENEO_EE_REPOSITORY }}"
```

Comment:
The same secret should be used in line 35

Reply:
Right, I split up the build in one of the past PRs.